modular_sojourn/general_things/perks.dm

```diff
@@ -0,0 +1,96 @@
+
+#define PERK_HUMAN_ARMOUR_BOOST /datum/perk/job/natural_armour
+/datum/perk/job/natural_armour
+	name = "Natural Rough Body"
+	desc = "You've been through EVERYTHING due to your frontier origins. Spider bites, random cuts, and slashes on rusted metal and wood, animal clawing and bitting. As a result, via natural adaptation using some sorta exoskeleton or just more resistant fur/skin/scales due to your ancestors, or simply you are stronger due to your rough body, you resist every type of damage just a little bit better than others due to your origins."
+
+/datum/perk/job/natural_armour/assign(mob/living/carbon/human/H)
+	..()
+	holder.brute_mod_perk -= 0.8
+	holder.burn_mod_perk -= 0.03
```

Comment:
3% reduction in burn is a bit odd, could be bit better like 5 or so to help with numbers